privacy/whonix.md

```diff
@@ -24,30 +24,25 @@ while the workstation is used for making AppVMs.
 
 ## Getting Started with Whonix
 
-To install Whonix, you must have a working Qubes machine already.
```

Comment:
It is pretty much the definition of obvious that you need a working Qubes installation to install Whonix inside your Qubes installation.